### DISCUSSION TITLE
Using other models besides openai's

### DISCUSSION THREAD

**Discussion by kenzabenjelloun** [microsoft/teams-sdk#182] [Q&A] [answered]
Hello ! 

I am trying to build a bot and i want to use a custom model from Azure ML, how can i do that please ? I can't seem to find the answer anywhere. 

Thank you !

**Comment by govil1811** [comment, marked as answer]
Hi @kenzabenjelloun,
This looks like an interesting use case. We currently offer default support OpenAI and Azure OpenAI. 
Details available: https://github.com/microsoft/teams-ai/tree/main/js/packages/teams-ai/src/OpenAIClients
If you have a custom LLM model please tweak and provide the connector file and use the existing integration for inspiration.

Once the custom model connector is available, you should be able to instantiate it and start building.

**Comment by corinagum** [comment]
We now have a custom model example for Llama in our samples! Please check it out.
